ui/components/app/terms-of-use-popup/terms-of-use-popup.js

```diff
@@ -0,0 +1,1155 @@
+          </Text>
+          <Text variant={TextVariant.bodyLgMedium} marginBottom={4}>
+            9. Disclaimers; Risk.
+          </Text>
+          <Text variant={TextVariant.bodySm} marginBottom={4}>
+            9.1 DISCLAIMER. THE OFFERINGS ARE PROVIDED “AS IS.” EXCEPT TO THE
+            EXTENT PROHIBITED BY LAW, OR TO THE EXTENT ANY STATUTORY RIGHTS
+            APPLY THAT CANNOT BE EXCLUDED, LIMITED OR WAIVED, WE AND OUR
+            AFFILIATES AND LICENSORS (A) MAKE NO REPRESENTATIONS OR WARRANTIES
+            OF ANY KIND, WHETHER EXPRESS, IMPLIED, STATUTORY OR OTHERWISE
+            REGARDING THE OFFERINGS OR THE THIRD-PARTY CONTENT, AND (B) DISCLAIM
+            ALL WARRANTIES, INCLUDING ANY IMPLIED OR EXPRESS WARRANTIES (I) OF
+            MERCHANTABILITY, SATISFACTORY QUALITY, FITNESS FOR A PARTICULAR
+            PURPOSE, NON-INFRINGEMENT, OR QUIET ENJOYMENT, (II) ARISING OUT OF
+            ANY COURSE OF DEALING OR USAGE OF TRADE, (III) THAT THE OFFERINGS OR
+            THIRD-PARTY CONTENT WILL BE UNINTERRUPTED, ERROR FREE OR FREE OF
+            HARMFUL COMPONENTS, AND (IV) THAT ANY CONTENT WILL BE SECURE OR NOT
+            OTHERWISE LOST OR ALTERED.
+          </Text>
+          <Text variant={TextVariant.bodySm} marginBottom={4}>
+            9.2 RISKS. OUR OFFERINGS RELY ON EMERGING TECHNOLOGIES, SUCH AS
+            ETHEREUM. SOME OFFERINGS ARE SUBJECT TO INCREASED RISK THROUGH YOUR
+            POTENTIAL MISUSE OF THINGS SUCH AS PUBLIC/PRIVATE KEY CRYPTOGRAPHY,
+            OR FAILING TO PROPERLY UPDATE OR RUN SOFTWARE TO ACCOMMODATE
+            PROTOCOL UPGRADES, LIKE THE TRANSITION TO PROOF OF STAKE CONSENSUS.
+            BY USING THE OFFERINGS YOU EXPLICITLY ACKNOWLEDGE AND ACCEPT THESE
+            HEIGHTENED RISKS.&nbsp; YOU REPRESENT THAT YOU ARE FINANCIALLY AND
+            TECHNICALLY SOPHISTICATED ENOUGH TO UNDERSTAND THE INHERENT RISKS
+            ASSOCIATED WITH USING CRYPTOGRAPHIC AND BLOCKCHAIN-BASED SYSTEMS AND
+            UPGRADING YOUR SOFTWARE AND PROCESSES TO ACCOMMODATE PROTOCOL
+            UPGRADES, AND THAT YOU HAVE A WORKING KNOWLEDGE OF THE USAGE AND
+            INTRICACIES OF DIGITAL ASSETS SUCH AS ETHER (ETH) AND OTHER DIGITAL
+            TOKENS, SUCH AS THOSE FOLLOWING THE ERC-20 TOKEN STANDARD.&nbsp; IN
+            PARTICULAR, YOU UNDERSTAND THAT WE DO NOT OPERATE THE ETHEREUM
+            PROTOCOL OR ANY OTHER BLOCKCHAIN PROTOCOL, COMMUNICATE OR EXECUTE
+            PROTOCOL UPGRADES, OR APPROVE OR PROCESS BLOCKCHAIN TRANSACTIONS ON
+            BEHALF OF YOU.&nbsp; YOU FURTHER UNDERSTAND THAT BLOCKCHAIN
+            PROTOCOLS PRESENT THEIR OWN RISKS OF USE, THAT SUPPORTING OR
+            PARTICIPATING IN THE PROTOCOL MAY RESULT IN LOSSES IF YOUR
+            PARTICIPATION VIOLATES CERTAIN PROTOCOL RULES, THAT&nbsp;
+            BLOCKCHAIN-BASED TRANSACTIONS ARE IRREVERSIBLE, THAT YOUR PRIVATE
+            KEY AND BACKUP SEED PHRASE MUST BE KEPT SECRET AT ALL TIMES, THAT
```

Comment:
We have used a lot of hardcoded copy here. Are we going to introduce the Terms of use only in English? As per the ticket, it's mentioned that it should be in 14 languages but I am unable to access the slack thread for this convo cc @kevinghim